boot/src/main/java/io/edpn/backend/application/controller/BootExplorationModuleController.java

```diff
@@ -0,0 +1,19 @@
+package io.edpn.backend.application.controller;
+
+import io.edpn.backend.exploration.application.controller.v1.DefaultExplorationModuleController;
+import io.edpn.backend.exploration.application.mappers.v1.SystemDtoMapper;
+import io.edpn.backend.exploration.domain.usecase.FindSystemsFromSearchbarUseCase;
+import io.edpn.backend.trade.application.controller.v1.DefaultTradeModuleController;
+import io.edpn.backend.trade.domain.service.v1.FindCommodityMarketInfoService;
+import io.edpn.backend.trade.domain.service.v1.FindCommodityService;
+import io.edpn.backend.trade.domain.service.v1.LocateCommodityService;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+public class BootExplorationModuleController extends DefaultExplorationModuleController {
+
+    public BootExplorationModuleController(FindSystemsFromSearchbarUseCase findSystemsFromSearchbarUseCase, SystemDtoMapper systemDtoMapper) {
+        super(findSystemsFromSearchbarUseCase, systemDtoMapper);
```

Comment:
"from search bar" - isn't that a FE concept? Should we not use the filters logic?

Reply:
the issue is that this is indeed an endpoint, that were it not for FE, I would not have created.

this is such a specific need to have performant and limited resultSet that I think it warants a specific endpoint